edgePy/DGEList.py

```diff
@@ -357,6 +357,48 @@ def rpkm(
 
         return self.copy(counts=counts, current_log=current_log, genes=genes)
 
+    def tpm(
```

Comment:
You should really do this in place of the original tpm() function, so we can compare with what was there before - it becomes much harder to compare and interpret your changes.

Reply:
Good point. I'll fix it tomorrow.